soc/xtensa/intel_adsp/common/include/cpu_init.h

```diff
@@ -68,11 +75,11 @@ static ALWAYS_INLINE void cpu_early_init(void)
 	 * bus.
 	 */
 	reg = 0x15;
```

Comment:
what is reg 0x15?

Reply:
ATOMCTL register control atomic operations. If I wanted to describe what this value mean I would probably need to add some definitions to the xtensa hal (at least I think so).

@softwarecki If I may ask, why are you reviewing code that doesn't change? :-P